core/src/test/scala/org/apache/spark/h2o/utils/SharedH2OTestContext.scala

```diff
@@ -49,7 +49,7 @@ trait SharedH2OTestContext extends SparkTestContext {
     try {
```

Comment:
What line could throw a ```SecurityException```?

Reply:
I believe `System.setSecurityManager(securityManager)`, would need to check, but it's not target of this PR :)

Reply:
Shouldn't be:
```
hc.stop()
hc = null
```
a part of finally statement then? Or is ```System.setSecurityManager(securityManager)``` a prerequisite for that?

Reply:
I think it's not target of this PR to discuss this particular part of code as nothing has been changed regarding this. Please create JIRA if something doesn't seem right and let's discuss & have a look later 👍

Reply:
but in short, yes. You're right!

Reply:
I'll add `hc = null` this line in this PR

Reply:
Yeah, np. :-) We can answer this question in a dedicate PR.

Reply:
Cool! Step by step :) I just want to avoid changing to much in one PR. But you raise good questions, thanks for that!